datadog_checks_downloader/tests/scripts/download_test_data.py

```diff
@@ -104,7 +104,11 @@ def zip_files(filename):
         for filename in filenames_for_target(signer_file, f'simple/datadog-{INTEGRATION}/index.html'):
             download_file(filename)
 
+        download_file(f'targets/simple/datadog-{INTEGRATION}/index.html')
```

Comment:
@cedricvanrompay-datadog Running this script without downloading the `index.html` or the actual wheel causes CI local download test to fail. Do you remember why it was passing without these beforehand?

Reply:
Thanks for the detailed response!